module/Activity/config/module.config.php

```diff
@@ -58,6 +58,56 @@
                                 'action' => 'create'
                             )
                         )
+                    ),
+                    'admin_queue' => array(
+                        'type' => 'Literal',
+                        'options' => array(
+                            'route' => '/admin/queue',
```

Comment:
Admin routes currently have the structure `/admin/<module>/.....`. Please adhere to this in here. Also, it is better to make all these routes that start with `/admin/` childs of a general route.